composer.json

```diff
@@ -26,7 +26,8 @@
     ],
     "require": {
         "yiisoft/yii2": "~2.0.0",
-        "psy/psysh": "~0.8.1"
+        "psy/psysh": "~0.8.1",
+        "symfony/var-dumper": "*"
```

Comment:
We should actually not use an unbound version constraint. But on the other hand ... they released a new version for three major versions 17 days ago :)

Reply:
~2.7|~3.0|~4.0